MLOps.NET.SQLite/Storage/LocalDbContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MLOps.NET.SQLite.Entities;
+
+namespace MLOps.NET.SQLite.Storage
+{
+    public class LocalDbContext : DbContext
```

Comment:
I'm not a SQLite expert, but would it make sense not to limit this to Local databases? E.g. I would image an end user probably wanting to have a central place for storage that may not be the same resource as the model is trained on.

Reply:
SQLite is file-based whereas MSSQL Server and MySQL are server-based. I think it is not meant for concurrence, therefore it is better to be local.
For the client-server scenario, we can use MSSQL, but in storage project.

Reply:
Gotcha, that was the missing piece of the puzzle, thanks!